src/main/resources/application.properties

```diff
@@ -48,9 +49,9 @@ closePaymentRetry.eventOffsetSeconds=${CLOSE_PAYMENT_RETRY_EVENT_BASE_INTERVAL_S
 closePaymentRetry.maxAttempts=${CLOSE_PAYMENT_RETRY_EVENT_MAX_ATTEMPTS}
 closePaymentRetry.paymentTokenValidityTimeOffset=${CLOSE_PAYMENT_RETRY_EVENT_PAYMENT_TOKEN_VALIDITY_TIME_OFFSET:10}
 
-transactionAuthorizationRequested.eventOffsetSeconds=${AUTHORIZATION_REQUESTED_RETRY_EVENT_BASE_INTERVAL_SECONDS}
-transactionAuthorizationRequested.maxAttempts=${AUTHORIZATION_REQUESTED_RETRY_EVENT_MAX_ATTEMPTS}
-transactionAuthorizationRequested.paymentTokenValidityTimeOffset=${AUTHORIZATION_REQUESTED_RETRY_EVENT_PAYMENT_TOKEN_VALIDITY_TIME_OFFSET:10}
+transactionAuthorizationOutcomeWaiting.eventOffsetSeconds=${AUTHORIZATION_OUTCOME_WAITING_EVENT_BASE_INTERVAL_SECONDS}
+transactionAuthorizationOutcomeWaiting.maxAttempts=${AUTHORIZATION_OUTCOME_WAITING_EVENT_MAX_ATTEMPTS}
+transactionAuthorizationOutcomeWaiting.paymentTokenValidityTimeOffset=${AUTHORIZATION_OUTCOME_WAITING_EVENT_PAYMENT_TOKEN_VALIDITY_TIME_OFFSET:10}
```

Comment:
can we explicit time unit for this time offset variable as done for `AUTHORIZATION_OUTCOME_WAITING_EVENT_BASE_INTERVAL_SECONDS`